packages/components/src/components/DeviceImage/index.tsx

```diff
@@ -1,56 +1,40 @@
+/* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
 import React from 'react';
-import styled from 'styled-components';
+import { DeviceModel } from '@trezor/device-utils';
 
 export interface DeviceImageProps {
-    trezorModel: 1 | 2;
+    deviceModel: DeviceModel;
     height?: string | number;
     hires?: boolean;
     className?: string;
 }
 
-const Image = styled.img``;
-
-const DeviceImage = ({ trezorModel, height = '100%', hires, className }: DeviceImageProps) => {
-    switch (trezorModel) {
-        case 1:
-            return (
-                <Image
-                    className={className}
-                    height={height}
-                    alt="trezor T1"
-                    src={
-                        hires
-                            ? require(`../../images/trezor/T1_hires.png`)
-                            : require(`../../images/trezor/T1.png`)
-                    }
-                    srcSet={
-                        hires
-                            ? `${require('../../images/trezor/T1_hires.png')} 1x, ${require('../../images/trezor/T1_hires@2x.png')} 2x`
-                            : undefined
-                    }
-                />
-            );
-        case 2:
-            return (
-                <Image
-                    className={className}
-                    height={height}
-                    alt="trezor T2"
-                    src={
-                        hires
-                            ? require(`../../images/trezor/T2_hires.png`)
-                            : require(`../../images/trezor/T2.png`)
-                    }
-                    srcSet={
-                        hires
-                            ? `${require('../../images/trezor/T2_hires.png')} 1x, ${require('../../images/trezor/T2_hires@2x.png')} 2x`
-                            : undefined
-                    }
-                />
-            );
-        // no default
+export const DeviceImage = ({
+    deviceModel,
+    height = '100%',
+    hires,
+    className,
+}: DeviceImageProps) => {
+    if (!deviceModel) {
+        return null;
     }
-};
 
-export { DeviceImage };
+    return (
+        <img
+            className={className}
+            height={height}
+            alt={deviceModel}
```

Comment:
Should I do a translation of 1 to `One`?

Reply:
We don't have to make the model names in alt attribute translatable, but it should be human readable, otherwise it's better not to define alt attribute at all

Reply:
Okay, I will leave it with `Trezor Model 1`